test/Microsoft.AspNetCore.Razor.Language.Test/SourceLocationTest.cs

```diff
@@ -61,7 +61,7 @@ public void GetHashCode_ReturnsSameValue_WhenEqual(string path)
         }
 
         [Fact]
-        public void Equals_ReturnsTrue_FilePathsNullAndAbsoluteIndicesMatch()
+        public void Equals_ReturnsFalse_FilePathsNullAndAbsoluteIndicesMatch()
```

Comment:
This test should be replaced with a good test.

Reply:
There is already a test for the True case